proposals/csharp-9.0/function-pointers.md

```diff
@@ -274,32 +274,32 @@ In unsafe code, the following changes are made to the type inference algorithms:
 
 #### Input types
 
-https://github.com/dotnet/csharplang/blob/master/spec/expressions.md#input-types
+[§11.6.3.4](https://github.com/dotnet/csharpstandard/blob/draft-v6/expressions.md#11634-input-types)
 
 The following is added:
 
 > If `E` is an address-of method group and `T` is a function pointer type then all the parameter types of `T` are input types of `E` with type `T`.
 
 #### Output types
 
-https://github.com/dotnet/csharplang/blob/master/spec/expressions.md#output-types
+[§11.6.3.5](https://github.com/dotnet/csharpstandard/blob/draft-v6/expressions.md#11635-output-types)
 
 The following is added:
 
 > If `E` is an address-of method group and `T` is a function pointer type then the return type of `T` is an output type of `E` with type `T`.
 
 #### Output type inferences
 
-https://github.com/dotnet/csharplang/blob/master/spec/expressions.md#output-type-inferences
+[§11.6.3.7](https://github.com/dotnet/csharpstandard/blob/draft-v6/expressions.md#11637-output-type-inferences)
 
 The following bullet is added between bullets 2 and 3:
 
 > * If `E` is an address-of method group and `T` is a function pointer type with parameter types `T1...Tk` and return type `Tb`, and overload resolution
 of `E` with the types `T1..Tk` yields a single method with return type `U`, then a _lower-bound inference_ is made from `U` to `Tb`.
 
-#### Exact inferences
+#### Better conversion from expression
 
-https://github.com/dotnet/csharplang/blob/master/spec/expressions.md#exact-inferences
+[§11.6.4.4](https://github.com/dotnet/csharpstandard/blob/draft-v6/expressions.md#11644-better-conversion-from-expression)
```

Comment:
It seems like the header and link were changed from "exact inferences" to "better conversion from expression". Is that intended?

Reply:
@jcouv Yes, that's intended. The standard doesn't have a header for "exact inferences". The text from that clause in the Microsoft spec is the first paragraph in the section "better conversion from expression" in the standard.